src/accelerate/utils/memory.py

```diff
@@ -174,7 +174,17 @@ def get_xpu_available_memory(device_index: int):
             from intel_extension_for_pytorch.xpu import mem_get_info
 
             return mem_get_info(device_index)[0]
+    elif version.parse(torch.__version__).release >= version.parse("2.6").release:
```

Comment:
That's Transformers function. For accelerate there is `is_torch_version` for the same purpose.

Unfortunately, at the moment I can't use these functions because pytorch 2.6 is not released yet and above functions don't give correct result for nightly 2.6 candidates (and for pytorch built from sources). Effectively they are doing this comparisons which render `False`:
```
$ python -c 'import torch; from packaging import version; print(torch.__version__)'
2.6.0.dev20241206+xpu
$ python -c 'import torch; from packaging import version; print(version.parse(torch.__version__) >= version.parse("2.6"))'
False
```

That's why I use explicit check comparing `.release` part of version strings. It handles above situation with nightly and from source builds:
```
$ python -c 'import torch; from packaging import version; print(version.parse(torch.__version__).release >= version.parse("2.6").release)'
True
```

I can update the check to use `is_torch_version()` once pytorch 2.6 will be available later in January. I hope this won't block merging this PR.

Reply:
Sounds good to me !